NetCoreRestApi/WebAPI/Controllers/CategoriesController.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Linq;
+using BusinessLayer.Manager;
+using Common.Converter;
+using DataLayer.Models;
+using Microsoft.AspNetCore.Mvc;
+using ServiceLayer.DataTransferObjects;
+
+namespace WebAPI.Controllers
+{  
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CategoriesController : ControllerBase
+    {
+        private IConverter<CategoryDTO, CategoryModel> _converter;
+        private ICategoryManager _categoryManager;
+
+        public CategoriesController(IConverter<CategoryDTO, CategoryModel> converter, ICategoryManager categoryManager)
+        {
+            _converter = converter;
+            _categoryManager = categoryManager;
+        }
+
+        [HttpGet]
+        public ActionResult<IQueryable<CategoryDTO>> Get()
+        {
+            var products = _categoryManager.GetAll();
+            if (products == null)
```

Comment:
this is redundant check. Manager should return empty list.

Reply:
I will remove that

Reply:
done